changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0.0
+
+- This is the first major stable release of Page Kit. N.B. It includes no changes since the last v0.x release.
```

Comment:
Non-blocking, could consider removing `n.b.` as it still makes sense without it.

Reply:
Pressed merge as this comment appeared.